pages/integrations/slack.md

```diff
@@ -25,6 +25,8 @@ Once you have granted access to your Slack workspace, give it a description, cho
 
 <%= image "buildkite-slack-connected.png", width: 1458/2, height: 1540/2, alt: "Screenshot of Buildkite Slack Notification Settings, requesting a description, your choice of text or emoji message themes, which pipelines and branches to include, and which build states should trigger a notification" %>
 
+The function of the "fixed" option displayed above is to notify you of a successful build of a pipeline after it has failed and will not be issued based on the actions of individual steps within a pipeline. The notifications configured above are at the pipeline level, not at the step level.
```

Comment:
This reads as if the failed build option causes it to be at the pipeline level, but I suspect that's not the case from the wording in the UI.

If it is:
```suggestion
With the configuration above, you'll receive notifications at the pipeline level. The _fixed builds_ option ensures you're notified when a failed build next passes, but not on the outcomes of individual steps within a pipeline.
```

If it's not:
```suggestion
With the configuration above, you'll receive notifications at the pipeline level but not on the outcomes of individual steps. The _fixed builds_ option ensures you're notified when a failed build next passes.
```

Reply:
I was trying to figure out where this code came from and I was really trying to think. Then I remembered this [PR](https://github.com/buildkite/docs/pull/1848) that was never looked at from months ago. This code actually belongs there, but if you're okay, we can merge it in here as well and close out the other PR.

"We should clarify what is meant by fixed as well as how it functions at the pipeline outcome level and not the step level."

So, it would be at the pipeline level, not the step level.

Reply:
Yeah, let's just include it here. So you can close the other PR when this one is merged :)